### PR TITLE
Fixed #114.

### LINKED ISSUE #114: 'Raw Preferences' does not reflect current preferences
Change a setting in preferences (say, indent width from 2 to 4), then open up 'Raw Preferences' and it will still list the setting as the old value (indentWidth = 2).

### PR DESCRIPTION
**Pull request by achesak**
Added win.save() calls to addTab() and closeTab().

The call in addTab() ensures that #114 is fixed, the call in closeTab() is to ensure the tabs are saved after a crash.

### CODE REVIEW

File: aporia.nim [hunk 1]
```diff
@@ -781,6 +782,9 @@ proc addTab(name, filename: string, setCurrent: bool = true,
   ## ``-1`` is returned upon error.
   assert(win.nimLang != nil)
 
+  # Save the preferences before the tab is opened.
+  win.save()
```
Comment (dom96) [inline aporia.nim]: Wouldn't this be more appropriate in the settings dialog (when it is closed)?

File: aporia.nim [hunk 2]
```diff
@@ -2080,7 +2084,11 @@ proc initsourceViewTabs() =
   if win.globalSettings.restoreTabs and lastSession.len > 0:
     for i in 0 .. lastSession.len-1:
       var splitUp = lastSession[i].split('|')
-      var (filename, offset) = (splitUp[0], splitUp[1])
+      var filename, offset: string
```
Comment (dom96) [inline aporia.nim]: what is this change for?

Comment (achesak) [inline aporia.nim]: An unrelated fix for when the config file doesn't have the proper format for previously open files, something I was messing around with and forgot to switch branches. I'll take it out next commit.

File: aporia.nim (hunk 1 above)
Comment (achesak) [inline aporia.nim]: Good point. I'll move that to after the tab is opened and add a new one in the preferences.

I'm a bit unfamiliar with how Nim does this though. The current 'win' var in settings.nim won't work as that's type ptr utils.MainWin. Setting a new variable and calling save() on that, without the pointer will compile fine but doesn't function. Is there another way to do this?

**Pull request merged**